docs/source/index.rst

```diff
@@ -0,0 +1,29 @@
+=============================
+Scylla Cluster Autoscaler Documentation
+=============================
+
+.. toctree::
+   :hidden:
+   :maxdepth: 2
+
+   generic
+   scylla_cluster_autoscaler_crd
+   recommender
+   updater
+   admission_controller
+
+Scylla Cluster Autoscaler is an open source project which helps users of Scylla Open Source and Scylla Enterprise run Scylla on Kubernetes (K8s)
```

Comment:
`
Scylla Cluster Autoscaler is an open source project which helps users of Scylla Open Source and Scylla Enterprise work with Scylla on Kubernetes (K8s).

The Scylla Cluster Autoscaler is an application capable of scaling Scylla database
clusters in an automated manner, freeing the human operator from the task of updating the
required specification manually. Using the elastic scale pattern, the Autoscaler manages to
scale the resources both vertically and horizontally.
Basic principle of SCA is that user defines a set of boolean queries on external,
performance-related metrics and the actions to be invoked, were their evaluated values true. It can be also be run in mode in which it produces recommendations, but does not invoke provided scaling actions.
SCA is also equipped with a safety mechanism which prevents sources other than SCA itself scaling controlled resources of a managed cluster.`

Would this be sufficient?